operators/src/source/ogr_source/mod.rs

```diff
@@ -525,6 +526,7 @@ where
             self.attribute_filters.clone(),
         )
         .await?
+        .merge_chunks(ctx.chunk_byte_size().into()) // rechunk the data if necessary TODO: remove when source produces the right chunk sizes
```

Comment:
Should we create a follow up task?

Reply:
yes it is the one with the builder byte size stuff.